server/routes/orion.js

```diff
@@ -376,8 +387,8 @@ exports.updateReview = function(req, res) {
               fixedReviewObject.reviewBody = utils.fixedEncodeURIComponent(
                 fixedReviewObject.reviewBody);
             }
-            utils.sendRequest('PATCH', fixedReviewObject, req.params.id,
-                req.headers)
+            utils.sendRequest('PATCH', fixedReviewObject, reviewId,
+                req.headers, {'options': 'keyValues'})
```

Comment:
too much nesting here, consider separate into functions and to create a more readable structure taking advantage of promises